### DISCUSSION TITLE
How can I send custom flags to fd with files?

### DISCUSSION THREAD

**Discussion by gengdz** [ibhagwan/fzf-lua#1451] [Q&A]
For example, I want to use the --glob option to search for files in a specified folder.

**Comment by ibhagwan** [comment]
```lua
require('fzf-lua').files({ cmd = 'fd <flags>' })
```

**Reply by gengdz** [reply]
I'm really sorry for not expressing myself clearly.

When I search for files, I want to include custom options.

For example, I want to search for files with the `.cjs` suffix：
![image](https://github.com/user-attachments/assets/8c5e09d2-ad18-434b-a1d9-abedc611ce2e)

**Reply by gengdz** [reply]
Just like when I use live grep, I can use `node -- -g '*.cjs'` to add custom parameters after the --.

![image](https://github.com/user-attachments/assets/e2f542ce-d202-4246-9960-14a246947d9f)

**Reply by ibhagwan** [reply]
No such option, files isn’t “live” the input prompt is used for fuzzy search only, it would be difficult to add since the query isn’t being parsed with each keystroke (as is with live variants).
